torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2463,6 +2486,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # TODO: remove redundant method_tests() entries
                SkipInfo('TestOpInfo', 'test_duplicate_method_tests')),
            sample_inputs_func=sample_inputs_addmm),
+    OpInfo('addmv',
+           dtypes=floating_types(),
+           dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
+           dtypesIfROCM=floating_types_and(torch.half),
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/55539 to track all issues for addmv
+               # failed with a large difference in tensor outputs for below tests
+               SkipInfo('TestCommon', 'test_variant_consistency_eager', dtypes=(torch.float32, torch.complex64)),
+               # AssertionError: UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
+               SkipInfo('TestCommon', 'test_out', dtypes=(torch.float32,)),
+               # RuntimeError: Jacobian mismatch for output 0 with respect to input 0,
+               # numerical:tensor([[ 1.0000e+00,  0.0000e+00,  0.0000e+00, -1.8758e+19,  0.0000e+00]])
+               # analytical:tensor([[1., 1., 1., 1., 1.]])
+               SkipInfo('TestGradients', 'test_inplace_grad', dtypes=(torch.float64,)),
+               # RuntimeError: Gradients failed to compare equal for grad output = 1j.
+               # Jacobian mismatch for output 0 with respect to input 0,
+               # numerical:tensor([[0.+1.0000e+00j, 0.+3.7808e+13j, 0.-9.1581e+12j, 0.+3.7517e+13j,
+               # 0.-1.5335e+13j]])
+               # analytical:tensor([[0.+1.j, 0.+1.j, 0.+1.j, 0.+1.j, 0.+1.j]])
+               SkipInfo('TestGradients', 'test_inplace_grad', dtypes=(torch.complex128,)),
+               # RuntimeError: "addmv_impl_cpu" not implemented for 'Half'
+               SkipInfo('TestCommon', 'test_inplace_grad', dtypes=(torch.complex128,)),
+               # RuntimeError: "addmv_impl_cpu" not implemented for 'Half'
+               SkipInfo('TestOpInfo', 'test_supported_backward', dtypes=(torch.float16,)),
+               # RuntimeError: "addmv_impl_cpu" not implemented for 'Half'
+               SkipInfo('TestOpInfo', 'test_supported_dtypes', dtypes=(torch.float16,)),
```

Comment:
I hope you wouldn't mind another unsolicited comment. :)
If `float16` isn't supported by `addmv` on CPU, then it should be removed from `dtypesIfCPU`.
BTW, you're skipping these 2 tests for CUDA as well, but the comments only mention CPU.

Reply:
The comments are very welcome-- I updated the tests!

Reply:
Hello @Lilyjjo, in my previous comment above, I had only been referring to the two tests above.
It seems that you'd have to skip `test_variant_consistency_eager` and `test_out` for 'CUDA' as well, so you might want to remove the device specifier in their respective `SkipInfo`s.